__tests__/e2e/06_Label.test.js

```diff
@@ -0,0 +1,48 @@
+import test from 'tape-async'
+import helper from 'tipsi-appium-helper'
+
+const { driver, select, idFromXPath } = helper
+
+test('<Label />', async (t) => {
+  const labelGroupId = select({
+    ios: idFromXPath(`
+      //XCUIElementTypeApplication/XCUIElementTypeWindow/XCUIElementTypeOther/
+      XCUIElementTypeOther/XCUIElementTypeOther/XCUIElementTypeOther/XCUIElementTypeOther/
+      XCUIElementTypeOther/XCUIElementTypeOther[2]/XCUIElementTypeOther/XCUIElementTypeOther/
+      XCUIElementTypeOther/XCUIElementTypeScrollView/XCUIElementTypeOther/XCUIElementTypeOther
+    `),
+    android: idFromXPath(`//
+      android.widget.LinearLayout[1]/android.widget.FrameLayout[1]/
+      android.widget.FrameLayout[1]/android.view.View[1]/android.view.View[2]/
+      android.view.View[1]/android.widget.ScrollView[1]/android.view.View[1]/
+      android.view.View
+    `),
+  })
+
+  const range = [
+    { id: 1, text: 'On Sale' },
+    { id: 2, text: 'Value Pick' },
+    { id: 3, text: 'Staff Pick' },
+  ]
+
+  try {
+    await helper.openExampleFor('<Label />')
+
+    for (const label of range) {
+      const id = `${labelGroupId}[${label.id}]`
+      const currentLabelId = select({
+        ios: `${id}/XCUIElementTypeOther[2]/XCUIElementTypeOther/XCUIElementTypeStaticText`,
+        android: `${id}/android.view.View[2]/android.widget.TextView[1]`,
```

Comment:
I think, you use _idFromXPath()_ function for building XPaths, because _currentLabelId_ path may not work on Android 6+